src/libs/OptionsListUtils.ts

```diff
@@ -2567,6 +2567,21 @@ function filterAndOrderOptions(options: Options, searchInputValue: string, confi
     };
 }
 
+/**
+ * Filter out selected options from personal details and recent reports
+ * @param options - The options to filter
+ * @param selectedOptions - The selected options to filter out. Keeping it as an array because it is unlikely to become big enough for Set to make a difference.
+ * @returns The filtered options
+ */
+function filterSelectedOptions(options: Options, selectedOptions: {accountID?: number}[]): Options {
+    const filteredOptions = {
```

Comment:
Was going to suggest using a `selectedAccountIDs: Set<number>` instead of `selectedOptions: Array<{accountID?: number}>`, but I see you _chose_ to keep it array because it's likely small enough that set won't make a difference.

Even if true, I'd encourage us to use a `Set` regardless, if only to set good examples of using the "correct" data structure when the only thing we're using an array for is looking up items in that array in a loop. The overhead of creating a Set is likely insignificant, but imo there's value in establishing and enforcing a pattern that sets should be used when appropriate for lookup-only structures.